devnet/node.yaml

```diff
@@ -75,6 +75,10 @@ spec:
             - ws://eth-devnet:8545
             - --avalancheRPC
             - ws://eth-devnet:8545
+           - --auroraRPC
```

Comment:
The indention on this line is off and it causes the build to fail since it makes it invalid.

YAML really cares about whitespace.